rollup.config.mjs

```diff
@@ -1,14 +1,9 @@
-import module from 'module'
 import path from 'path'
 import commonjs from '@rollup/plugin-commonjs'
 import {nodeResolve} from '@rollup/plugin-node-resolve'
 import {terser} from 'rollup-plugin-terser'
 
-// eslint-disable-next-line node/no-deprecated-api -- Remove when `@rollup/plugin-babel` supports ESM.
-var requireUtil = module.createRequireFromPath(
-  path.join(process.cwd(), 'rollup.config.mjs')
-)
-var babel = requireUtil('@rollup/plugin-babel').babel
+import {babel} from '@rollup/plugin-babel'
```

Comment:
Did you try to run test? I have no idea why CI passed this but babel plugin does not have es modules support. `module` field is only for bundlers. Node does not use it.

Reply:
Please revert.

Reply:
Oh, I see. It's not babel that supports es modules. But node now can have named imports from commonjs. And it's backported to latest node 12 and 14. Very interesting.

Reply:
Wait that sounds interesting @TrySound, what did you find out?

Reply:
I had a few month old node which failed with this change. After update it worked.
Looks like node team implemented named exports detection in commonjs https://github.com/nodejs/node/pull/35249